### PR TITLE
task/secure-tokens

### PR DESCRIPTION
**Pull request by eisenbergrobin**
Stores tokens in UserProperties instead of as a function parameter. 
Uses a prompt to request the token from the user. 

Screenshot 2019-03-15 at 22 05 36

Maintains retro-compatibility (not very cleanly might I add) for the old signature.

### CODE REVIEW

**Comment by eisenbergrobin** [comment]
Review just https://github.com/botify-labs/botify-integration-google-sheets/pull/2/commits/600f1b0677ebeb2dc86ea6e735a1b465a0446d75
and
https://github.com/botify-labs/botify-integration-google-sheets/pull/2/commits/35de407bf4e7e84329f2d42a99e32f552d98bd9c

File: index.gs [hunk 1]
```diff
@@ -0,0 +1,62 @@
+function getTokenFromProperties() {
+  return PropertiesService.getUserProperties().getProperty("BOTIFY_API_TOKEN");
+}
+
+function init() {
+  SpreadsheetApp.getUi()
+    .createMenu("Botify Addon")
+    .addItem("Edit authentication        ", "promptForApiToken")
```
Comment (rgehan) [inline index.gs]: What's all that space for?

**Review by rgehan — COMMENTED** [review]
Nothing strikes me, it LGTM and feels way less hacky than the previous arguments matching method you used.

I just have one probably irrelevant question.

**Comment by zouzouek** [comment]
LGTM

File: index.gs (hunk 1 above)
Comment (eisenbergrobin) [reply index.gs]: To make it look good ©️ 

No really, if I don't add the space the menu looks nasty...

**Pull request merged**